src/core_modules/capture-core/components/QuickSelector/Program/CategorySelector.component.js

```diff
@@ -82,6 +82,7 @@ class CategorySelector extends React.Component<Props, State> {
             prevOrgUnitId: null,
         };
         this.onSelectSelector = makeOnSelectSelector();
+        // todo you cannot setState from this component (lgtm report)
```

Comment:
Don't remember why I did this from the top of my head. Migth be worth looking into at some point.